ntbs-service/Pages/Notifications/Edit/HospitalDetails.cshtml.cs

```diff
@@ -133,6 +134,10 @@ protected override async Task ValidateAndSave()
                 nameof(Notification),
                 Notification.NotificationDate,
                 nameof(Notification.NotificationDate));
+            if (Notification.HospitalDetails.CaseManagerId == HospitalDetails.CaseManagerId)
+            {
+                ModelState.Remove("HospitalDetails.CaseManagerId");
+            }
```

Comment:
For anyone looking at the code, this validation rule will be very difficult to spot, because it isn't defined in the same place as all the other validation rules (i.e. on the `HospitalDetails`).

I think a better way to do this is to add a property to `HospitalDetails` called something like `ExistingCaseManagerId` which is set during `SetValuesForValidation`. Then, you can change the validation rules on the `CaseManagerId` in the `HospitalDetails` to reflect the fact that the `ExistingCaseManagerId` is always an allowed value.

Reply:
Made this change